fontforge/parsettf.c

```diff
@@ -4558,6 +4558,30 @@ static int PickCMap(struct cmap_encs *cmap_encs,int enccnt,int def) {
 return( ret );
 }
 
+/* This doesn't worry about putting the most appropriate encoding
+   in sc->unicodeenc because (in theory) NameConsistancyCheck()
+   will swap it in. */
+static void addttfencoding(SplineChar *sc, int unc) {
+    struct altuni *alt;
+    if ( unc==-1 ) {
+	return;
+    } else if (sc->unicodeenc==-1 || sc->unicodeenc==unc) {
```

Comment:
Seems like you could move `sc->unicodeenc==unc` to the if statement above

Reply:
done